frontend/webservice/monitoring.cgi

```diff
@@ -248,5 +258,48 @@ sub send_json {
     print "Content-type: text/plain\n\n" . encode_json($output);
 }
 
+sub get_oess_status {
+    my $result;
+    my $serviceResult;
+    
+    return if !defined $mq;
+
+    $mq->{'topic'} = 'MPLS.Discovery.RPC';
+    $mq->{'timeout'} = 2;
+    $serviceResult = $mq->is_online();
+    if (defined $serviceResult->{error}) {
+        push(@$result,{ServiceName => 'Discovery', Status => 'Down'});
```

Comment:
Convention is to use snake_case for variable names using all lowercase letters. I would prefer the value of status to be lowercase as well; This will be used on the frontend programmatically so keeping the options simple is desirable.

Reply:
Fixed ServiceName to be service_name, and all services to their lower case counterparts and Up and Down to up and down